Utils/RandomUtils.js

```diff
@@ -79,4 +79,59 @@ function isChance(percentage) {
 
 function random() {
     return arguments[randomInteger(0, arguments.length - 1)];
+}
+
+function waitForDone() {
```

Comment:
Already exists: https://github.com/GodDragoner/Spicy-TAJ-Personality/blob/681cd985ad177de79f126b35f003dc53a91f2cdc/Session/Modules/Pain/SmallPunishment.js#L221

Reply:
yes, but 1) thats a wierd place for it, and 2) the added timeout behavior breaks legacy code that assumes infinite timeout.